### PR TITLE
Xaa s

### PR DESCRIPTION
**Pull request by wombatpiobair**
Give this a hard look for grammar and the like. There was a reported rendering issue that I was not able to reproduce.

### CODE REVIEW

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# 3.0.1 Cloud Service Ecosystems
```
Comment (zjorge96) [inline 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:1]: This is a great addition to the bootcamp. I'd recommend just moving the information from 3.0.1 over into "3.0-overview.md".

File: 3-cloud-computing/3.1-cloud.md [hunk 2]
```diff
@@ -2,13 +2,18 @@
 
 ### Cloud Computing Platforms
 
-Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. 
+Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. Cloud prividers
+
```
Comment (zjorge96) [inline 3-cloud-computing/3.1-cloud.md:6]: Small misspelling with "prividers". Also, remove the extra newlines so it reads as

"Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. Cloud providers also offer varying ecosystems of IaaS, CaaS, PaaS and FaaS.

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+to the user as an integrated solution, solution stack, or service through an internet connection.
+
+Primarily useful for developers and programmers, PaaS allows the user to develop, 
+run, and manage their own apps without having to build and maintain the 
+infrastructure or platform usually associated with the process. External resources, such as
+databases, and queues are configured and attached to runtime containers by the platform. 
+Developers can use built-in provisioned resources to create their applications.
+
+The consumer writes, builds, and manages their apps, but they do it without the headaches of 
+software updates, hardware maintenance or containerization. The consumer pushes an uncontainerize
```
Comment (zjorge96) [inline 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:59]: Possible misspelling with "uncontainerize"? Possibly "uncontainerized"?

File: _sidebar.md [hunk 4]
```diff
@@ -31,6 +31,7 @@
   - [2.5.3 - Deliverables](2-virtual-machines-containers/2.5.3-deliverables.md)
 * **Chapter 3**
 - [3.0 - Cloud Computing](3-cloud-computing/3.0-overview.md)
+  - [3.0.1 - Cloud Service Ecosystems](3-cloud-computing/3.0.1-Cloud Service Ecosystems.md)
```
Comment (zjorge96) [inline _sidebar.md:34]: If you decide to keep 3.0.1, due to the spaces in the file name "3.0.1-Cloud Service Ecosystems.md" it is causing it to error out a bit.  I attached a photo of the section in the sidebar below.

Screenshot 2023-02-17 at 11 27 19 AM

**Review by zjorge96 — CHANGES_REQUESTED** [review]
Added some comments. Just needs some slight tweaks, but overall it looks great!

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
+# 3.0.1 Cloud Service Ecosystems
+
+![](./img3/cloudservices.png )
+
+There are many different types of cloud service ecosystems all delivering varying levels of management for applications, data and
+their underlying infrastructure. They are as follows:
+
+## Self-Managed
```
Comment (twellspring) [inline 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:8]: I would add in this section that it is not a cloud service but included to show the increasing level of outsourcing.  Want to include co-location in this section?

File: 3-cloud-computing/3.1-cloud.md (hunk 2 above)
Comment (wombatpiobair) [reply 3-cloud-computing/3.1-cloud.md:6]: fixed

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md (hunk 5 above)
Comment (wombatpiobair) [reply 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:8]: good comment - resolved

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md (hunk 3 above)
Comment (wombatpiobair) [reply 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:59]: changed to un-containerized

File: _sidebar.md (hunk 4 above)
Comment (wombatpiobair) [reply _sidebar.md:34]: changed filename and removed spaces

File: 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md (hunk 1 above)
Comment (wombatpiobair) [reply 3-cloud-computing/3.0.1-Cloud Service Ecosystems.md:1]: It fit better as it's own subsection

**Review by zjorge96 — APPROVED** [review]

**Pull request merged**